libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -33,6 +32,39 @@ const AP_Param::GroupInfo AP_Mount::var_info[] = {
     AP_GROUPEND
 };
 
+void AP_Mount::Write_Mount(enum LogMessages msg, uint64_t timestamp_us)
```

Comment:
As mentioned this should take an instance and then we should be logging the angles for just that instance.